pr-data.csv

```diff
@@ -3598,12 +3598,12 @@ https://github.com/apache/shenyu,ae6ddc5abe3e88f1c0a5a9201a59bcdb74a04157,shenyu
 https://github.com/apache/shenyu,ae6ddc5abe3e88f1c0a5a9201a59bcdb74a04157,shenyu-admin,org.apache.shenyu.admin.listener.websocket.WebsocketDataChangedListenerTest.testOnRuleChanged,ID,Opened,https://github.com/apache/shenyu/pull/6229,
 https://github.com/apache/shenyu,ae6ddc5abe3e88f1c0a5a9201a59bcdb74a04157,shenyu-admin,org.apache.shenyu.admin.listener.websocket.WebsocketDataChangedListenerTest.testOnSelectorChanged,ID,Opened,https://github.com/apache/shenyu/pull/6229,
 https://github.com/apache/shenyu,22546ae0e6a4047ee08bf4a41f9114f47ba57e19,shenyu-admin,org.apache.shenyu.admin.service.PermissionServiceTest.testGetPermissionMenu,ID,Accepted,https://github.com/apache/shenyu/pull/5241,
-https://github.com/apache/shenyu,22546ae0e6a4047ee08bf4a41f9114f47ba57e19,shenyu-admin,org.apache.shenyu.admin.service.register.ShenyuClientRegisterDivideServiceImplTest.testBuildHandle,ID,Accepted,https://github.com/apache/shenyu/pull/4040,
-https://github.com/apache/shenyu,22546ae0e6a4047ee08bf4a41f9114f47ba57e19,shenyu-admin,org.apache.shenyu.admin.service.register.ShenyuClientRegisterDubboServiceImplTest.testBuildHandle,ID,Accepted,https://github.com/apache/shenyu/pull/4043,
-https://github.com/apache/shenyu,22546ae0e6a4047ee08bf4a41f9114f47ba57e19,shenyu-admin,org.apache.shenyu.admin.service.register.ShenyuClientRegisterGrpcServiceImplTest.testBuildHandle,ID,Accepted,https://github.com/apache/shenyu/pull/4043,
+https://github.com/apache/shenyu,22546ae0e6a4047ee08bf4a41f9114f47ba57e19,shenyu-admin,org.apache.shenyu.admin.service.register.ShenyuClientRegisterDivideServiceImplTest.testBuildHandle,ID,Opened,https://github.com/apache/shenyu/pull/6232,https://github.com/apache/shenyu/pull/4040
```

Comment:
Change the note to provide more info, e.g., `https://github.com/apache/shenyu/pull/4040 was previously accepted but does not work any more`